src/coffee/constants.coffee

```diff
@@ -44,7 +44,7 @@ constants =
 
   ATTRIBUTE_CONSTRAINT_SAME_FOR_ALL: 'SameForAll'
 
-  REGEX_PRICE: new RegExp /^(([A-Za-z]{2})-|)([A-Z]{3}) (-?\d+)(-?\|(\d+)|)( ([^~\$#]*)|)(#([^~\$]*)|)(\$([^~]*)|)(~(.*)|)$/
+  REGEX_PRICE: new RegExp /^(([A-Za-z]{2})-|)([A-Z]{3}) (-?\d+)(-?\|(\d+)|)( ([^~\$#%]*)|)(#([^~\$%]*)|)(\$([^~%]*)|)(~([^%]*)|)(%([A-Z]{3} -?\d+ @\d+(.*))|)$/
```

Comment:
Add tiers format to the price regex and defined the symbol (`%`) to be used for tiers.

Reply:
Woah, this is insane 😄